README.md

```diff
@@ -1,81 +1,72 @@
 
-This Turborepo has some additional tools already setup for you:
+The development server runs at `https://localhost:3333`. If this port is taken, the project will error.
 
-- [TypeScript](https://www.typescriptlang.org/) for static type checking
-- [ESLint](https://eslint.org/) for code linting
-- [Prettier](https://prettier.io) for code formatting
+## Workspace Setup
 
-### Build
+This repo used pnpm, a space efficient replacement to npm.
+You can learn more about it's advantages at <https://pnpm.io/>.
 
-To build all apps and packages, run the following command:
+Install pnpm with the following command, or use an [alternative instalation method](https://pnpm.io/installation).
 
-```
-cd my-turborepo
-pnpm build
+```bash
+npm install -g pnpm
 ```
 
-### Develop
+When running commands, use `pnpm` rather than `npm`.
 
-To develop all apps and packages, run the following command:
+### Dependencies
 
-```
-cd my-turborepo
-pnpm dev
+Running the following command will install dependencies for all apps and packages regardless of where it's executed.
+
+```bash
+pnpm install
 ```
 
-### Remote Caching
+When installing a new dependency for an individual app or package, first navigate to the corresponding directory, then run the `pnpm add` command.
 
-Turborepo can use a technique known as [Remote Caching](https://turbo.build/repo/docs/core-concepts/remote-caching) to share cache artifacts across machines, enabling you to share build caches with your team and CI/CD pipelines.
+If you want to install a package shared by multiple apps or packages, run the command at the project root. You'll have to use the -w flag (or --workspace-root) to confirm your intention.
 
-By default, Turborepo will cache locally. To enable Remote Caching you will need an account with Vercel. If you don't have an account you can [create one](https://vercel.com/signup), then enter the following commands:
+### Build
 
-```
-cd my-turborepo
-npx turbo login
+To build all apps and packages, run the following command in the project root.
+
+```bash
+pnpm build
 ```
 
-This will authenticate the Turborepo CLI with your [Vercel account](https://vercel.com/docs/concepts/personal-accounts/overview).
+To build an individual app or package, first navigate to the corresponding directory and run the build command.
 
-Next, you can link your Turborepo to your Remote Cache by running the following command from the root of your Turborepo:
+### Develop
 
+To develop all apps and packages, run the following command in the project root.
+
+```bash
+pnpm dev
 ```
-npx turbo link
-```
 
-## Useful Links
+To develop an individual app or package, first navigate to the corresponding directory and run the dev command.
+
+## License
 
-Learn more about the power of Turborepo:
+This repo is currently source available and not licensed under an open source license. This may be subject to change in the future.
```

Comment:
Suggestion: would "source available ~and~**but** not licensed" make more sense here?